shell/pages/diagnostic.vue

```diff
@@ -0,0 +1,439 @@
+      <h2 class="mb-20">
+        {{ t('about.diagnostic.logs.subtitle') }}
+      </h2>
+      <ul class="logs-container">
+        <li
+          v-for="logEntry in latestLogs"
+          :key="generateKey(logEntry.timestamp)"
+          :class="logEntry.type"
+        >
+          <span class="log-entry-type">{{ logEntry.type }}: </span>
+          <span
+            v-for="(arg, i) in logEntry.data"
+            :key="i"
+          >{{ arg }}</span>
+        </li>
+      </ul>
+    </div>
+    <div class="mb-40">
+      <h2 class="mb-20">
+        {{ t('about.diagnostic.resourceCounts.subtitle') }}
+      </h2>
+      <div class="resources-count-container">
+        <table
+          v-for="cluster in finalCounts"
+          :key="cluster.id"
+          class="full-width"
+        >
+          <thead @click="toggleTable(cluster.id)">
+            <th colspan="2">
+              <div>
+                <span>{{ cluster.id }}</span>
+                <i
+                  class="icon"
+                  :class="{
+                    'icon-chevron-down': !cluster.isTableVisible,
+                    'icon-chevron-up': cluster.isTableVisible
+                  }"
+                ></i>
+              </div>
+            </th>
+          </thead>
+          <tbody v-show="cluster.isTableVisible">
+            <tr v-for="item in cluster.counts" :key="item.resource">
+              <td>{{ item.resource }}</td>
+              <td>{{ item.count }}</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.title-block {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  h1 {
+    margin: 0 !important;
+  }
+
+  div {
+    display: flex;
+    align-items: center;
+  }
+}
+
+table {
+  border-collapse: collapse;
+  overflow: hidden;
+  border-radius: var(--border-radius);
+
+  &.full-width {
+    border-radius: 0 !important;
+    min-width: 60%;
+
+    thead {
+      cursor: pointer;
+
+      th div {
+          display: flex;
+          align-items: center;
+          justify-content: space-between;
+      }
+    }
+  }
+
+  tr > td:first-of-type {
+    width: 20%;
+  }
+
+  th, td {
+    border: 1px solid var(--border);
+    padding: 8px 5px;
+    min-width: 150px;
+    text-align: left;
+  }
+
+  th {
+    background-color: var(--sortable-table-top-divider);
+    border-bottom: 1px solid var(--sortable-table-top-divider);
+  }
+
+  a {
+    cursor: pointer;
+  }
+
+  .os {
+    display: flex;
+    align-items: center;
+  }
+}
+
+.logs-container {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  background-color: #FFF;
+  border: 1px solid #000;
+  height: 300px;
+  overflow-x: hidden;
+  overflow-y: auto;
+
+  li {
+    font-family: 'Courier New', monospace;
+    font-size: 13px;
+    margin: 0;
+    padding: 10px 20px;
+    border-bottom: 1px solid #ccc;
+
+    &.warn {
+      background-color: LightGoldenRodYellow;
```

Comment:
just a comment (pretty sure you got this already, but just in case), generally all colours should be standard but as we're forcing a white background and making the content look like the `console` these are fine